m2cgen/interpreters/visual_basic/interpreter.py

```diff
@@ -0,0 +1,63 @@
+import os
+
+from m2cgen import ast
+from m2cgen.interpreters import mixins, utils
+from m2cgen.interpreters.interpreter import ToCodeInterpreter
+from m2cgen.interpreters.visual_basic.code_generator \
+    import VisualBasicCodeGenerator
+
+
+class VisualBasicInterpreter(ToCodeInterpreter, mixins.LinearAlgebraMixin):
+    supported_bin_vector_ops = {
+        ast.BinNumOpType.ADD: "addVectors",
+    }
+
+    supported_bin_vector_num_ops = {
+        ast.BinNumOpType.MUL: "mulVectorNumber",
+    }
+
+    exponent_function_name = "Math.Exp"
+    tanh_function_name = "Tanh"
+
+    def __init__(self, module_name="Model", indent=4, *args, **kwargs):
+        self.module_name = module_name
+        cg = VisualBasicCodeGenerator(indent=indent)
+        kwargs["feature_array_name"] = "input_vector"
```

Comment:
Just wondering is there any reason why we can't use the existing default name?

Reply:
Please see this https://github.com/BayesWitnesses/m2cgen/pull/109#discussion_r336280288.

Reply:
Ah, sorry, missed that.

Reply:
No problem! Some languages can successfully distinguish when `input` is a built-in function and when it isn't, e.g. Python, but VBA cannot do that 🙁 . 
But I think that the best practice is to not name variables with reserved keywords (`print`, `input`, `file`, etc.) in **all** languages.